src/dex-helper/dummy-dex-helper.ts

```diff
@@ -146,6 +146,10 @@ class DummyCache implements ICache {
   }
 
   async hget(mapKey: string, key: string): Promise<string | null> {
+    if (!this.hashStorage[mapKey]) {
+      return null;
+    }
+
     return this.hashStorage[mapKey][key];
```

Comment:
this would ensure we never return `undefined`, wdyt?
```suggestion
    return this.hashStorage[mapKey]?.[key] ?? null;
```

Reply:
@nzrii-kysel  fixed